playbooks/common/openshift-cluster/config.yml

```diff
@@ -45,6 +45,10 @@
   tags:
   - hosted
 
+- include: service_catalog.yml
```

Comment:
should we use that var for opting in? currently I'm using the following in the roles:
```yaml
openshift_service_catalog_remove=True
openshift_service_catalog_broker_remove=True
```

I'm not sure if they are mutually exclusive or not,  @pmorie ?